themes/envy/envy.theme.bash

```diff
@@ -6,11 +6,24 @@ SCM_THEME_PROMPT_SUFFIX="${green}|"
 
 GIT_THEME_PROMPT_DIRTY=" ${red}✗"
 GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
-GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_PREFIX=" ${green}| "
```

Comment:
A [Search of the repository](https://github.com/Bash-it/bash-it/search?p=1&q=GIT_THEME_PROMPT_PREFIX&unscoped_q=GIT_THEME_PROMPT_PREFIX) shows that, by and large, themes like this one use the original form:
```
GIT_THEME_PROMPT_PREFIX=" ${green}|"
```

I think it would be better if you moved this change to your local `.bash_profile` and leave this default as-is to match the general usage.

Reply:
Yes, i revert is in recent PR.